lib/elixir/test/elixir/stream_test.exs

```diff
@@ -168,12 +168,16 @@ defmodule StreamTest do
     stream = Stream.cycle([1, 2, 3])
     assert is_function(stream)
 
-    assert_raise ArgumentError, fn ->
+    assert_raise ArgumentError, "cannot cycle over empty enumerable", fn ->
       Stream.cycle([])
     end
 
-    assert_raise ArgumentError, fn ->
-      Stream.map([], &(&1)) |> Stream.cycle |> Enum.to_list
+    assert_raise ArgumentError, "cannot cycle over empty enumerable", fn ->
+      Stream.map([], &(&1)) |> Stream.cycle() |> Enum.to_list()
+    end
+
+    assert_raise ArgumentError, "cannot cycle over empty enumerable", fn ->
+      Stream.map(%{}, &(&1)) |> Stream.cycle() |> Enum.to_list()
```

Comment:
I actually meant that instead of using `Stream.map/2` on an empty list to create a stream, an empty map would have worked just fine as a stream :) Sorry I wasn't clearer!